index.bs

```diff
@@ -822,7 +823,7 @@ The <dfn method for=FileSystemDirectoryHandle>getFileHandle(|name|, |options|)</
      abort these steps.
 
   1. Let |entry| be the result of [=locating an entry=] given |locator|.
-  1. If |options|.{{FileSystemGetFileOptions/create}} is true:
+  1. If |options|'s {{FileSystemGetFileOptions/create}} is true:
```

Comment:
Dictionaries can be accessed as Infra maps. So I think this should be:

> `|options|["{{FileSystemGetFileOptions/create}}"]`

Reply:
Ah thanks for the clarification. Updated all the dictionary accesses to follow this pattern